userspace/libsinsp/filter.cpp

```diff
@@ -1294,9 +1308,294 @@ void sinsp_filter::pop_expression()
 	m_nest_level--;
 }
 
+//
+// Check to see if the filter has 1 or more  SQL 'in' clause(s)
+// return modified filter to "or" clauses, else return the original filter string
+// throw if no filter field before SQL "in" delimitor, or no "," after multiple values
+//
+string sinsp_filter::parse_sql_in_clause(const string& fltstr)
```

Comment:
I would just call this parse_in_clause. I think using the word sql is misleading